inc/composer/class-plugin.php

```diff
@@ -107,7 +107,24 @@ public function generate_module_manifest() {
 			$module_loader .= "\n// Load {$package->getName()}.\nrequire_once __DIR__ . '{$file}';";
 		}
 
+		// Get custom module entrypoints.
+		$root_extra = $this->composer->getPackage()->getExtra();
+
+		if ( isset( $root_extra['altis']['modules'] ) ) {
+			foreach ( $root_extra['altis']['modules'] as $module => $config ) {
+				if ( ! isset( $config['entrypoint'] ) ) {
+					continue;
+				}
+
+				$files = array_filter( (array) $config['entrypoint'], 'file_exists' );
+				$files = implode( "';\nrequire_once dirname( __DIR__ ) . DIRECTORY_SEPARATOR . '", $files );
```

Comment:
Ooh, I just noticed how this is working; could we `array_map` instead, then `implode( "\n" )`? It's a little unclear as-is, and I'd definitely be confusing coming back to it.

Reply:
Yep, you'd definitely be confusing 😬